src/atomistics/calculators/lammps/helpers.py

```
@@ -48,11 +49,11 @@ def lammps_run(
 
 
 def lammps_calc_md_step(
-    lmp_instance,
+    lmp_instance: LammpsASELibrary,
     run_str: str,
     run: int,
-    output_keys=OutputMolecularDynamics.keys(),
-):
+    output_keys: list[str] = OutputMolecularDynamics.keys(),
```

Ruff (0.15.14)

[warning] Do not perform function call `OutputMolecularDynamics.keys` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)